### PR TITLE
feat: adds basic helm chart with deployment and service

### PR DESCRIPTION
**Pull request by mindovermiles262**
## Description

Adds a basic Helm Chart for deploying Password Pusher

## Related Issue

None

## Type of Change

- [ ] 📚 Examples / docs / tutorials / dependencies update
- [ ] 🔧 Bug fix (non-breaking change which fixes an issue)
- [x] 🥂 Improvement (non-breaking change which improves an existing feature)
- [x] 🚀 New feature (non-breaking change which adds functionality)
- [ ] 💥 Breaking change (fix or feature that would cause existing functionality to change)
- [ ] 🔐 Security fix

## Checklist

- [x] I've written tests (if applicable) for all new methods and classes that I created. (`rake test`)
- [x] I've added documentation as necessary so users can easily use and understand this feature/fix.

### CODE REVIEW

File: containers/helm/values.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    enabled: true
+    path: /
+  livenessProbe:
+    enabled: true
+    path: /
+
+image:
+  repository: docker.io/pglombardo/pwpush-ephemeral
+  pullPolicy: Always
+  tag: "latest"
```
Comment (pglombardo) [inline containers/helm/values.yaml:22]: Let's point this to `release` since that's mostly stable.  `latest` breaks fairly often.

**Comment by pglombardo** [comment]
Hi @mindovermiles262 - thanks for the great contribution!  Looks great.  I added one comment.  Also If you feel inclined, you can add a "helm" section to the front README as well.   Otherwise I'll add it myself after merge.

Could you point the container image to `release` tag instead?  It's the stable tag.

**Comment by mindovermiles262** [comment]
Suggested changes made. Please let me know if you have anything else

Thanks

**Comment by pglombardo** [comment]
Thanks @mindovermiles262!

**Pull request merged**